tech/languages/csharp/mono-installation.md

```diff
@@ -1,24 +1,44 @@
 ---
-title: Mono
+title: C#
 subsection: csharp
 section: tech-languages
-order: 1
+order: 4
 version: 7.8.4
-description: "C# provided by Mono: a cross-platform, open source, .NET development framework"
+description: "C# - .NET Core and Mono"
 ---
 
 # Mono installation
 
-Fedora comes with [Mono](http://www.mono-project.com/) -- a free and open source implementation of the .Net framework. To install it, simply type:
+Fedora comes with [Mono](http://www.mono-project.com/) -- a free and open source implementation of the .NET Framework. To install it, simply type:
 
 ```
 $ sudo dnf install mono-devel
 ```
 
 This command will install Mono -- the runtime environment and associated development tools.
 
-[MonoDevelop](http://www.monodevelop.com/) is the official IDE for development with C# on Mono. To install it, simply type:
+
+# NAnt installation
+
+The most popular build tool for Mono is [NAnt](http://nant.sourceforge.net/). To install it, simply type:
+
+```
+$ sudo dnf install nant
+```
+
+# NUnit installation
+
+[NUnit](http://nunit.org/) is very useful for test driven development. To install it, simply type:
 
 ```
-$ sudo dnf install monodevelop
+$ sudo dnf install nunit nunit-gui
```

Comment:
Because i'm not mono contributor and know nothing of it. They received an email but did not update their pages.